kolibri/core/assets/src/composables/useConnection.js

```diff
@@ -0,0 +1,17 @@
+import { ref } from 'kolibri.lib.vueCompositionApi';
+import Vue from 'vue';
+import VueCompositionApi from '@vue/composition-api';
+
+Vue.use(VueCompositionApi);
```

Comment:
You shouldn't need to use the VueCompositionApi in the individual composable, as we do that on a global level, so I am fairly sure this should work without this.

Reply:
I'm not sure why but I get failing tests if I don't include this. Are you able to reproduce this?

Reply:
Fixed